### PR TITLE
Docs for Kedro-Viz CLI 

### LINKED ISSUE #1705: Add CLI reference
## Description
It would be nice to have a CLI reference for `kedro viz` subcommands.

## Context
I was wondering what the output of `kedro viz run --save-file` should be, so I went to the documentation. But the docs explain usage, there is no reference docs (related: https://github.com/kedro-org/kedro/issues/2604)

## Possible Implementation
https://sphinx-click.readthedocs.io/

image

## Possible Alternatives
...

## Checklist

- [ ] Include labels so that we can categorise your feature request

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Resolves #1705 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by astrojuanlu — APPROVED** [review]
Much better than not having anything ❤️ 

@rashidakanchwala Did you look at https://sphinx-click.readthedocs.io/en/latest/ ? (Instead of writing the docs manually)

**Comment by rashidakanchwala** [comment]
> Much better than not having anything ❤️
> 
> @rashidakanchwala Did you look at https://sphinx-click.readthedocs.io/en/latest/ ? (Instead of writing the docs manually)

Nicee, technically I didn't write it. ChatGPT did, i just cleaned it up :) -- but I will look at the above.

File: docs/source/cli-docs.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+```bash
+kedro viz deploy --platform aws --endpoint http://mybucket.s3-website-us-west-2.amazonaws.com --bucket-name mybucket
+```
+
+### Building Kedro Viz to host on multiple platforms 
+
+To create a build directory with the visualisation data:
+
+```bash
+kedro viz build --include-hooks
```
Comment (ravi-kumar-pilla) [inline docs/source/cli-docs.md:158]:
```suggestion
kedro viz build --include-previews
```

File: docs/source/cli-docs.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+**Options:**
+
+- `--host <host>`
+  - Host that Kedro Viz will listen to. Defaults to `localhost`.
+  
+- `--port <port>`
+  - TCP port that Kedro Viz will listen to. Defaults to `4141`.
+
+- `--browser / --no-browser`
+  - Whether to open the Kedro Viz interface in the default browser. The browser will open if the host is `localhost`. Defaults to `True`.
```
Comment (ravi-kumar-pilla) [inline docs/source/cli-docs.md:44]: Do we need this here ? - The browser will open if the host is `localhost`

**Review by ravi-kumar-pilla — APPROVED** [review]
Left a minor comment. LGTM 👍 ... thank you @rashidakanchwala

File: docs/source/cli-docs.md (hunk 2 above)
Comment (rashidakanchwala) [reply docs/source/cli-docs.md:44]: Yes, i got it from the docs. Maybe we can reword it in cli, and then change it here.

**Pull request merged**